scripts/bigquery.js

```diff
@@ -0,0 +1,267 @@
+require('dotenv').config()
+const { BigQuery } = require('@google-cloud/bigquery')
+const { getMongo, connect } = require('../db')
+const { collections } = require('../services/constants')
+const { getAdvancedReports } = require('../services/sentry/analyticsRecorder')
+
+const REPORT_PUBLISHER_TO_ADUNIT_TABLE_NAME = 'reportPublisherToAdUnit'
+const REPORT_PUBLISHER_TO_COUNTRY_TABLE_NAME = 'reportPublisherToCountry'
+const BIGQUERY_RATE_LIMIT = 10 // There is a limit of ~ 2-10 min between delete and insert or changing schema
```

Comment:
Where did you get this data? I have seen claims of almost 2 hours between delete and insert on stackoverflow

Reply:
I found this on stackoverflow but sometimes I needed more then 2 min for the data to import:
https://stackoverflow.com/a/25292028

As for the update with query I have seen that there might be limits so I did not even try it:
https://cloud.google.com/bigquery/quotas#standard_tables